Maquetado/views/login.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Login</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="../application.css">
+</head>
+
+<body>
+  <div class='pageContainer'>
+    <div class='loginContainer'>
+      <img src='../resources/img/LogoWolox.png' class='woloxLoginLogo' alt='Wolox Logo Image' />
+      <div class='inputsContainer'>
+        <h2 class='dataTitle'>Email</h2>
+        <input class='dataInput'>
+        <h2 class='dataTitle'>Password</h2>
+        <input class='dataInput'>
```

Comment:
Add types to the inputs.
```suggestion
        <input class='dataInput' type='text'>
        <h2 class='dataTitle'>Password</h2>
        <input class='dataInput' type='email'>
```